src/applications/Applications.ExtractImages/ExtractImagesHost.cs

```diff
@@ -56,12 +58,10 @@ public class ExtractImagesHost : MicroserviceHost
             //                      to a helper class to support having multiple configurations (and probably prevent some bugs)
             string extractionName = _fileSystem.Path.GetFileNameWithoutExtension(_csvFilePath);
             string extractionDir = _fileSystem.Path.Join(cliOptions.ProjectId, "extractions", extractionName);
-            string absoluteExtractionDir = _fileSystem.Path.Join(extractRoot, extractionDir);
-
-            if (_fileSystem.Directory.Exists(absoluteExtractionDir))
-                throw new DirectoryNotFoundException($"Extraction directory already exists '{absoluteExtractionDir}'");
+            _absoluteExtractionDir = _fileSystem.Path.Join(extractRoot, extractionDir);
 
-            _fileSystem.Directory.CreateDirectory(absoluteExtractionDir);
+            if (_fileSystem.Directory.Exists(_absoluteExtractionDir))
+                throw new DirectoryNotFoundException($"Extraction directory already exists '{_absoluteExtractionDir}'");
```

Comment:
Surely if the file is empty that would be ok? Is this to prevent multiple extractions overwritting each other?

Reply:
Yep, this is to prevent any file clobbering.